apiserver/app/createInvoice_test.go

```diff
@@ -77,3 +76,36 @@ func TestCreateInvoice(t *testing.T) {
 		t.Errorf("Response was incorrect. We expected %s", invoice)
 	}
 }
+
+func TestRateLimiter(t *testing.T) {
+	t.Cleanup(cleanDb(t))
+	a.cache.FlushAll(ctx)
+	_, sessionToken := createTestUser(t, "")
+
+	var requestBody bytes.Buffer
+	multipartWriter := multipart.NewWriter(&requestBody)
+	multipartWriter.WriteField("format", entity.UblFormat)
+	multipartWriter.WriteField("test", "true")
+
+	invoiceWriter, _ := multipartWriter.CreateFormFile("invoice", "ubl21_invoice.xml")
+	invoice, _ := ioutil.ReadFile("../../xml/ubl21/example/ubl21_invoice.xml")
+	invoiceWriter.Write(invoice)
+	multipartWriter.Close()
+	body := requestBody.Bytes()
+
+	for i := 0; i < 20; i++ {
```

Comment:
let's set upper limit in test environment to lower number let's say `1`, so tests are faster